controllers/socket/lobbyHandlers.go

```diff
@@ -359,15 +394,20 @@ func playerReadyHandler(so socketio.Socket) func(string) string {
 				return string(bytes)
 			}
 
+			helpers.LockRecord(lobby.ID, lobby)
```

Comment:
Instead of doing it this way, just lock the lobby at the start of the function (or, better, the first use of lobby) and unlock it after its last use. Locks are more expensive compared to ordinary calculations and there's no need to lock and unlock stuff multiple times in a function